packages/openactive-broker-microservice/src/state.js

```diff
@@ -126,24 +187,21 @@ const state = {
 };
 
 /**
+ * All Opportunity IDs that are considered "locked" (because they have already
+ * been used in a test) for the specified [Test Dataset](https://openactive.io/test-interface/#datasets-endpoints).
+ *
  * @param {string} testDatasetIdentifier
+ * @returns {Set<string>}
  */
-function getTestDataset(testDatasetIdentifier) {
-  if (!state.testDatasets.has(testDatasetIdentifier)) {
-    state.testDatasets.set(testDatasetIdentifier, new Set());
+function getLockedOpportunityIdsInTestDataset(testDatasetIdentifier) {
+  if (!state.lockedOpportunityIdsByTestDataset.has(testDatasetIdentifier)) {
+    state.lockedOpportunityIdsByTestDataset.set(testDatasetIdentifier, new Set());
   }
-  return state.testDatasets.get(testDatasetIdentifier);
+  return state.lockedOpportunityIdsByTestDataset.get(testDatasetIdentifier);
 }
 
-function getAllDatasets() {
-  return new Set(Array.from(state.testDatasets.values()).flatMap((x) => Array.from(x.values())));
-}
-
-/**
- * @param {string} feedIdentifier
- */
-function addFeed(feedIdentifier) {
```

Comment:
`addFeed(..)` seemed very ambiguous, so this has been replaced with `state.incompleteFeeds.markFeedHarvestStarted(..)`